lib/ansible/plugins/action/__init__.py

```diff
@@ -763,7 +762,7 @@ def _execute_module(self, module_name=None, module_args=None, tmp=None, task_var
 
     def _remove_internal_keys(self, data):
         for key in list(data.keys()):
-            if key.startswith('_ansible_') or key in C.INTERNAL_RESULT_KEYS:
+            if key.startswith('_ansible_') and key != '_ansible_parsed' or key in C.INTERNAL_RESULT_KEYS:
```

Comment:
well, parsed should be only key needed at this point, anything else should be consumed inside the previous code or generated afterwards